rose/server/score.py

```diff
@@ -40,27 +40,26 @@ def process(players, track):
             if player.action != actions.JUMP:
                 track.clear(player.x, player.y)
                 player.y += 1
-                player.score += config.score_move_backward
+                player.score += config.score_move_backward * 2
             else:
                 player.score += config.score_jump
-
         elif obstacle in (obstacles.TRASH,
                           obstacles.BIKE,
                           obstacles.BARRIER):
-            track.clear(player.x, player.y)
-            player.y += 1
-            player.score += config.score_move_backward
+            if player.action != actions.LEFT and player.action != actions.RIGHT:
```

Comment:
LEFT and RIGHT are already handled. Why do we need to consider 
them again?

Reply:
The condition is on line 49, before the handling that starts on line 50.

Reply:
@nirs Before the change, for bike, trash and barrier, there was no check for the action chosen by the driver. Each time those obstacles were met the driver was losing points and staying in place (since the track moves forward, and the driver moves down, he actually stays in place).
I added the condition and moved the actions under it.